source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -18,36 +18,64 @@ Wallets
 
 |cryptox| for Android
 ---------------------
-    **1.1.1 - June 11, 2024**
+    **1.2.0 - August 27, 2024**
 
-    Version 1.1.1 improves WalletConnect pairing stability in situations where the connection is poor
+    This update introduces support for Company ID, CCD listings, and a newsfeed, along with optional anonymous analytics. It also fixes several visual bugs.
```

Comment:
Should we use anonymous?

Reply:
I don't see an issue here, this sentence doesn't mention any crypto like it was in the dialog message

Reply:
Ok.